media-libs/amdgpu-pro-vulkan/amdgpu-pro-vulkan-22.10.4.1452060-r1.ebuild

```diff
@@ -115,20 +118,59 @@ src_install() {
 		insinto /usr/share/vulkan/icd.d
 		doins "${S}"/"${PN}"-i386/opt/amdgpu-pro/etc/vulkan/icd.d/amd_pro_icd32.json
 	fi
+
+	# AMD Vulkan Prefixes
+	newbin "${WORKDIR}/amd-vulkan-prefixes-main/amd_vulkan_prefixes.sh" vk_radv
+	newbin "${WORKDIR}/amd-vulkan-prefixes-main/amd_vulkan_prefixes.sh" vk_amdvlk
+	newbin "${WORKDIR}/amd-vulkan-prefixes-main/amd_vulkan_prefixes.sh" vk_pro
+
+	newbashcomp "${WORKDIR}/amd-vulkan-prefixes-main/amd_vulkan_prefixes.bash-completion" vk_radv
+	newbashcomp "${WORKDIR}/amd-vulkan-prefixes-main/amd_vulkan_prefixes.bash-completion" vk_amdvlk
+	newbashcomp "${WORKDIR}/amd-vulkan-prefixes-main/amd_vulkan_prefixes.bash-completion" vk_pro
+
+	if use use-as-default ; then
+		#set AMDGPU-PRO ICD loader system wide
+		doenvd "${FILESDIR}/99-amdgpu-pro-icd"
+	else
+		#set AMDGPU (MESA) ICD loader system wide
+		doenvd "${FILESDIR}/99-amdgpu-icd"
```

Comment:
What if the user has both Nvidia and AMD GPUs, here we force to use Mesa thus the AMD one. So we definitely shouldn't do this